primitives/src/lib.rs

```diff
@@ -177,3 +177,26 @@ pub enum ReserveIdentifier {
 pub type NFTBalance = u128;
 
 pub type CashYieldIndex = u128;
+
+/// Increase decimals
+pub fn convert_decimals_inc(b: Balance) -> Balance {
```

Comment:
The name is a bit confusing. Maybe `convert_decimals_to_evm` and `convert_decimals_from_evm`

And comment needs to explain this convert decimal between native (12) and EVM (18) and therefore the `1_000_000` conversion.